### PR TITLE
Update to Numba 0.49.1

### PR DESCRIPTION
**Pull request by PokhodenkoSA**
Also removed mentions of 0.48.0. I hope I have found all places.

### CODE REVIEW

**Comment by Vyacheslav-Smirnov** [comment]
@PokhodenkoSA , also need to set 0.49.0 in requirements.txt

File: setup.py [hunk 1]
```diff
@@ -316,7 +316,7 @@ def run(self):
           'numpy>=1.16',
           'pandas==0.25.3',
           'pyarrow==0.15.1',
-          'numba>=0.48'
+          'numba >=0.49.0,<0.50.0'
```
Comment (AlexanderKalistratov) [inline setup.py:319]: Why not 0.49.1?

Comment (PokhodenkoSA) [reply setup.py:319]: For not be binded to 0.49.1 when 0.49.2 will be released.

Comment (AlexanderKalistratov) [reply setup.py:319]: So then probably it would be better `> 0.49.0` ?

Comment (PokhodenkoSA) [reply setup.py:319]: B/c 0.50.0 could break backward compatibility.

Comment (AlexanderKalistratov) [reply setup.py:319]: `< 0.49.0, <0.50.0`

Comment (PokhodenkoSA) [reply setup.py:319]: But 0.49.0 is also workable.

Comment (PokhodenkoSA) [reply setup.py]: Numba 0.49.0 has significant regressions, it is better not to use it and start from 0.49.1.

**Pull request merged**